src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -27033,6 +27680,62 @@ void Validate(ModuleSymbol m)
                 symbolValidator: Validate);
         }
 
+        [Fact]
+        public void PropertyImplementationInDerived_15()
+        {
+            var source1 =
+@"
+public interface I2
+{
+    virtual int M1 {get => throw null; private set => throw null;} 
+}
+
+public interface I4
+{
+    int M1 {set => throw null; private get => throw null;} 
+}
+
+public interface I5 : I4
+{
+}
+
+public interface I1 : I2, I5
+{
+    int I2.M1
+    {
+        get
+        {
+            System.Console.WriteLine(""I2.M1"");
+            return 1;
+        }
+    }
+    int I4.M1 
+    {
+        set => System.Console.WriteLine(""I4.M1"");
+    }
+}
+
+public interface I3 : I1
```

Comment:
>I3 [](start = 17, length = 2)

nit: `I3` seems unused

Reply:
> nit: I3 seems unused

Its presence is a use. It doesn't target any specific problem, but doesn't hurt, I believe.

---
In reply to: [257399182](https://github.com/dotnet/roslyn/pull/33413#discussion_r257399182) [](ancestors = 257399182)